tests/commons/web/test_parse_headers.py

```
@@ -0,0 +1,70 @@
+    ],
+)
+def test_parse_headers_errors(
+    mode: str,
+    content: str,
+    expected_error: str | None,
+    expected_result: tuple[str, dict[str, str]] | None,
+) -> None:
+    if expected_error:
+        with pytest.raises(ValueError, match=expected_error):
+            parse_header(content, mode)
+            raise AssertionError("There should have been an error")
```

Ruff (0.8.2)

`pytest.raises()` block should contain a single simple statement

(PT012)